CHANGELOG.md

```diff
@@ -19,6 +19,7 @@ Changes
 * [UI]: Fixed: Allow organism name not included in the current taxonomy.
 * [Developer]: Removed `prop-types` as a front-end development dependency.
 * [UI]: Updated `User` and `Admin` help documentation URLs.
+* [UI]: Updated message displayed to user when an invalid reference file is uploaded for a SNVPhyl workflow.
```

Comment:
Can you update this to remove reference to SNVPhyl since it is the only workflow that requires a reference and had this issue. Maybe replace `for a SNVPhyl workflow.` to `while launching a workflow.` (or something generic like that)

Reply:
Updated in 43238ff7dcd9b3bfcd1dba48a7f4b2efd07d1607